examples/stable_diffusion_docker.yaml

```diff
@@ -0,0 +1,35 @@
+
+# Steps to run this example:
```

Comment:
This is great! Can we also put them in a README.md? Can even add Step 0: `pip install skypilot` and point to our Installation docs page. Maybe some title like "Run your own Stable Diffusion UI in the cloud with one click"?

```
examples/stable_diffusion/
  README.md
  this.yaml
  Dockerfile
```

Goal is for people to come to this dir, look at README and immediately get started.

Reply:
+1, we should move this to `examples/stable_diffusion`